Source/Workers/createVerticesFromQuantizedTerrainMesh.js

```diff
@@ -85,6 +99,22 @@ define([
             if (hasVertexNormals) {
                 toPack.x = octEncodedNormals[n];
                 toPack.y = octEncodedNormals[n + 1];
+
+                if (exaggeration !== 1.0) {
+                    var normal = AttributeCompression.octDecode(toPack.x, toPack.y, new Cartesian3());
```

Comment:
Are you sure we don't want to use a scratch here and below for the `Matrix4`?  This is per vertex on tile load, right?